tools/topology/topology2/development/CMakeLists.txt

```diff
@@ -23,6 +23,15 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-adl-nocodec.bin,DEEPBUFFER_FW_DMA_MS=1
 # SSP topology for MTL
 "cavs-nocodec\;sof-mtl-nocodec\;PLATFORM=mtl,NUM_DMICS=2,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-nocodec.bin,DEEPBUFFER_FW_DMA_MS=100"
+
+# CAVS HDA topology with mixer-based fir eq pipelines for HDA and passthrough pipelines for HDMI
+"sof-hda-generic\;sof-hda-fir-generic\;HDA_CONFIG=fir,USE_CHAIN_DMA=true,DEEPBUFFER_FW_DMA_MS=100,EFX_FIR_PARAMS=passthrough"
+"sof-hda-generic\;sof-hda-fir-generic-2ch\;\
```

Comment:
@juimonen @singalsu why is this a separate tplg? With IPC3 we had this in the regular HDA topologies isnt it?

Reply:
ok, I'll move to draft and move to generic